ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -6,179 +6,274 @@
 import UIKit
 import SwiftUI
 
-/// Pre-defined styles of the avatar
-@objc public enum MSFAvatarStyle: Int, CaseIterable {
-    case `default`
-    case accent
-    case group
-    case outlined
-    case outlinedPrimary
-    case overflow
-}
+/// Design token set for the `Avatar` control.
+open class AvatarTokens: ControlTokens {
+    // MARK: - Design Tokens
 
-/// Pre-defined sizes of the avatar
-@objc public enum MSFAvatarSize: Int, CaseIterable {
-    case xsmall
-    case small
-    case medium
-    case large
-    case xlarge
-    case xxlarge
+    /// The size of the content of the `Avatar`.
+    open var avatarSize: CGFloat {
+        return size.size
```

Comment:
I think this is odd, but wasn't really a fan of having the redundant code from the size. There's a few places throughout the codebase where we get the size of an avatar straight from the enum, though, so I didn't want to break those.

Reply:
Can you open a bug to remove `size` from `MSFAvatarSize` and move it here? I tried fixing it locally and only `LargeTitleView` is giving me a bit of a headache.